### ISSUE TITLE
Failures for 2019-03-05 pre-prod XML 7 of 9

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#1521]
Pre-prod SF-86 7 of 9:
https://drive.google.com/open?id=1S-vQUPyeRPGCpp6OEacHwksmjZEMPR1w

Exceptions when submitted to e-QIP:
```
WARN[0004] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S09-Citizenship: Do not provide a country for "Documentation of U.S. Citizen Born Abroad: Place of issuance" when selecting a state or territory. 
WARN[0004] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S13A-EmploymentUnemploymentInformation: Provide a response for "Supervisor telephone number". 
WARN[0004] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S14-SelectiveServiceRegistration: Answer "Have you registered with the Selective Service System (SSS)?" question yes, no or I don't know. 
```

**Comment by ryanhofdotgov** [comment]
Looks like one of the errors is also in https://github.com/18F/e-QIP-prototype/issues/1516

**Comment by ryanhofdotgov** [comment]
Exception 1 addressed by #1594, Exception 2 is a dup of #1516, and Exception 3 is resolved by #1596.
